test/powershell/Modules/Microsoft.PowerShell.Utility/Invoke-Item.Tests.ps1

```diff
@@ -37,14 +37,58 @@ Describe "Invoke-Item" -Tags "CI" {
         $testfile = Join-Path $TestDrive testfile.txt
     }
 
-    It "Should invoke a text file without error" -Skip:($IsWindows -and $IsCoreCLR) {
+    It "Should invoke a text file without error on non-Windows" -Skip:($IsWindows) {
         $debugfn = NewProcessStartInfo "-noprofile ""``Invoke-Item $testfile`n" -RedirectStdIn
         $process = RunPowerShell $debugfn
         EnsureChildHasExited $process
         $process.ExitCode | Should Be 0
     }
 
+<#
     It "Should throw not supported on Windows with .NET Core" -Skip:($IsLinux -or $IsOSX -or !$IsCoreCLR) {
         { Invoke-Item $testfile }| Should Throw "Operation is not supported on this platform."
     }
+#>
+}
+
+Describe "Invoke-Item tests on Windows" -Tags "CI","RequireAdminOnWindows" {
+    BeforeAll {
+        $testfilename = "testfile.!!testext!!"
+        $testfilepath = Join-Path $TestDrive $testfilename
+        $renamedtestfilename = "renamedtestfile.!!testext!!"
+        $renamedtestfilepath = Join-Path $TestDrive $renamedtestfilename
+        remove-item $testfilepath -ErrorAction SilentlyContinue
+        remove-item $renamedtestfilepath -ErrorAction SilentlyContinue
+        new-item $testfilepath | Out-Null
+        if ( $IsWindows -and -not ($IsWindows -and $IsCoreCLR) ) {
```

Comment:
This is an equivalent of `$IsWindows -and -not $IsCoreCLR`.
BTW, why do you exclude coreclr build from it?